cloudify_cli/commands/replace_certificates.py

```diff
@@ -70,17 +70,18 @@ def start_replace_certificates(input_path,
                                force,
                                logger):
     _validate_username_and_private_key()
-    # output_path is not a default param because of `cfy.options.output_path`
     config_dict = get_dict_from_yaml(_get_input_path(input_path))
     logger.info('Validating replace-certificates config file...')
     validate_config_dict(config_dict, force, logger)
 
     main_config = ReplaceCertificatesConfig(config_dict, False, logger)
     main_config.validate_certificates()
+    logger.info('\nReplacing certificates...')
```

Comment:
Why the leading newline?

Reply:
Just to separate it from the validations outputs

Reply:
Fair enough.